apps/guide/src/content/03-popular-topics/02-threads.mdx

```diff
@@ -0,0 +1,222 @@
+
+Threads are automatically archived after inactivity. "Activity" is defined as sending a message, unarchiving a thread, or changing the auto-archive time.
+
+To archive or unarchive a thread, use the <DocsLink type="class" parent="ThreadChannel" symbol="setArchived" brackets /> method and pass in a boolean parameter:
+
+<CH.Code>
+
+```js focus=2,3
+const thread = channel.threads.cache.find((x) => x.name === 'food-talk');
+await thread.setArchived(true); // Archived.
+await thread.setArchived(false); // Unarchived.
+```
+
+</CH.Code>
+
+This same principle applies to locking and unlocking a thread via the <DocsLink type="class" parent="ThreadChannel" symbol="setLocked" brackets /> method:
+
+<CH.Code>
+
+```js focus=2,3
+const thread = channel.threads.cache.find((x) => x.name === 'food-talk');
+await thread.setLocked(true); // Locked.
+await thread.setLocked(false); // Unlocked.
+```
+
+</CH.Code>
+
+## Public and private threads
+
+Public threads are viewable by everyone who can view the parent channel of the thread. Public threads can be created with the <DocsLink type="class" parent="GuildTextThreadManager" symbol="create" brackets /> method.
+
+<CH.Code>
+
+```js
+import { ThreadAutoArchiveDuration } from 'discord.js';
+
+const thread = await channel.threads.create({
+	name: 'food-talk',
+	autoArchiveDuration: ThreadAutoArchiveDuration.OneHour,
+	reason: 'Needed a separate thread for food',
+});
+
+console.log(`Created thread: ${thread.name}`);
+```
+
+</CH.Code>
+
+They can also be created from an existing message with the <DocsLink type="class" parent="Message" symbol="startThread" brackets /> method, but will be "orphaned" if that message is deleted.
+
+<CH.Code>
+
+```js focus=3[22:42]
+import { ThreadAutoArchiveDuration } from 'discord.js';
+
+const thread = await message.startThread({
+	name: 'food-talk',
+	autoArchiveDuration: ThreadAutoArchiveDuration.OneHour,
+	reason: 'Needed a separate thread for food',
+});
+
+console.log(`Created thread: ${thread.name}`);
+```
+
+</CH.Code>
+
+The created thread and the message it originated from will share the same id. The type of thread created matches the parent channel's type.
+
+Private threads behave similar to Group DMs, but in a Guild. Private threads can only be created on text channels.
+
+To create a private thread, use <DocsLink type="class" parent="GuildTextThreadManager" symbol="create" brackets /> and pass in _`ChannelType.PrivateThread`_ as the _`type`_:
+
+<CH.Code>
+
+```js focus=1[10:21],6
+import { ChannelType, ThreadAutoArchiveDuration } from 'discord.js';
+
+const thread = await channel.threads.create({
+	name: 'mod-talk',
+	autoArchiveDuration: ThreadAutoArchiveDuration.OneHour,
+	type: ChannelType.PrivateThread,
+	reason: 'Needed a separate thread for moderation',
+});
+
+console.log(`Created thread: ${thread.name}`);
+```
+
+</CH.Code>
+
+## Adding and removing members
+
+You can add and remove members to and from a thread channel.
```

Comment:
Obvious part, I think it should be deleted

Reply:
Its useful but should have more info about the permissions required